### PR TITLE
Docs/clean up rendering docs

### PR DESCRIPTION
**Pull request by abegong**
This PR cleans up the top-level ToC, and refines the text in the first three sections (Introduction, Getting started, Community.)

ToC currently looks like this:

screencapture-file-Users-abe-Documents-superconductive-tools-great-expectations-docs-build-html-index-html-2019-08-01-10_53_04

### CODE REVIEW

**Comment by abegong** [comment]
Ready for review. I've thoroughly revised the `Introduction` and `Community` sections.

I've also revised the top-level `Getting started` page, but not the internal content.

Double-checked to verify that links from notebooks are still intact.

Dropped the extraneous README files in the `render/` directory.

File: docs/intro.rst [hunk 1]
```diff
@@ -21,18 +21,53 @@ Software developers have long known that automated testing is essential for \
...
+
+**Automated data profiling**
+
+	Writing pipeline test from scratch can be tedious and counterintuitive. Great Expectations jump starts the process by providing powerful tools for automated data profiling. This provides the double benefit of helping you explore data faster, and capturing knowledge for future documentation and testing.
+
+**DataContexts and DataSources**
+
+	...allow you to configure connections your data stores, using names you’re already familiar with: “the ml_training_results bucket in S3,” “the Users table in Redshift.” Great Expectations provides convenience libraries to introspect most common data stores (Ex: SQL databases, data directories and S3 buckets.) We are also working to integrate with pipeline execution frameworks (Ex: airflow, dbt, dagster, prefect.io). The Great Expectations framework lets you fetch, validate, profile, and document your data in a way that’s meaningful within your existing infrastructure and work environment.
```
Comment (eugmandel) [inline docs/intro.rst:59]: Is the "..." intentional?

File: docs/intro.rst [hunk 2]
```diff
@@ -45,36 +80,39 @@ It's not unusual to see data teams grind to a halt for weeks (or even months!) t
...
 -------------------------------------------------------------
 
-It depends. If you have needs that the library doesn't meet yet, please
-`upvote an existing
-issue(s) <https://github.com/great-expectations/great_expectations/issues>`__
-or `open a new
-issue <https://github.com/great-expectations/great_expectations/issues/new>`__
-and we'll see what we can do. Great Expectations is under active
-development, so your use case might be supported soon.
+Great Expectations is under active development by James Campbell, Abe Gong, Eugene Mandell and Rob Lim, with help from many others.
```
Comment (eugmandel) [inline docs/intro.rst]: "Mandell"->"Mandel"

**Review by eugmandel — CHANGES_REQUESTED** [review]

File: docs/intro.rst (hunk 1 above)
Comment (abegong) [reply docs/intro.rst:59]: yes, since it's a continuation of the header.

File: docs/intro.rst (hunk 2 above)
Comment (abegong) [reply docs/intro.rst]: Oops--sorry about that.

**Review by eugmandel — APPROVED** [review]

**Pull request merged**